### DISCUSSION TITLE
No drainage on latest update and duplicate sensors

### DISCUSSION THREAD

**Discussion by milsky99** [jeroenterheerdt/HAsmartirrigation#486] [Q&A]
I have updated to v 8.0 although inside HACS its showing 8.2 yet no drainage options for my zones, this is quite important as my drainage is very poor and watering often is adjusted rather by the multiplier which isnt ideal. I have created a new zone to test and this is also not showing drainage.

Second issue is that it appears as if my zones sensors are being duplicated in HA, no idea why this is happening but the above test zone created 2 additional ones.

**Comment by jeroenterheerdt** [comment]
First off, clear your browser cache please. Let me know if after clearing cache and restarting you are still having issues.
